packages/dds/matrix/src/test/time/.mocharc.cjs

```diff
@@ -0,0 +1,20 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+/**
+ * Mocha configuration file to run execution-time-profiling tests
+ */
+
+module.exports = {
+	"exit": true,
```

Comment:
Do things work correctly without this? If we need `exit: true` in our mocha config files or `--exit` as a flag, that's a sign of a problem (potentially unrelated to the tests, but if possible we should avoid it).

Reply:
It runs well for just this directory benchmark tests. Not sure if this configures setting used by ci tests